.gitignore

```diff
@@ -1,3 +1,6 @@
 vendor/*
 config.yml
 composer.phar
+
+# Library don't need composer.lock on repo
+composer.lock
```

Comment:
@igormukhingmailcom I think we do need the composer.lock to keep prod env exactly the same as dev